service/src/test/resources/ehr/mapper/encountercomponents/expected-components-1.xml

```diff
@@ -1,29 +1,30 @@
-<component typeCode="COMP">
+<component typeCode="COMP" >
```

Comment:
Were all of these trailing spaces added by running the IDE formatter or similar? If so please revert, it makes it difficult to see the relevant changes for this feature.